packages/vscode-extension/src/extension/main.ts

```diff
@@ -110,10 +139,22 @@ function registerOnDidOpenTextDocListener(
       ),
     );
 
-    vscode.window.showInformationMessage(
-      "'.pliplugin' folder and files created successfully.",
+    // confirmation message
+    vscode.window.showInformationMessage(CONFIG_CREATION.SUCCESS_MESSAGE);
+  }
+}
+
+function registerOnDidChangeActiveTextEditor(
+  telemetryReporter: TelemetryReporter | undefined,
+) {
+  const listener = vscode.window.onDidChangeActiveTextEditor(async (editor) => {
+    await handleMissingConfig(
+      editor,
+      telemetryReporter,
+      "pli.language.support.documentAtivated",
```

Comment:
Typo on `documentAtivated`, and not sure if this one was requested specifically? If so then we should correct it, but if not then I would probably remove it.